JFSD.py

```diff
@@ -1,10 +1,12 @@
 import numpy as np
 
-from jfsd import main, utils
+from jfsd import main, mainBD, utils
```

Comment:
Module names should not have capital letters. I haven't changed anything for the current ones, so that I wouldn't break anything, but for new models we should not break the "rules".

Reply:
the module mainBD is not needed anymore [https://github.com/torrewk/Python-Jax-Fast-Stokesian-Dynamics/pull/13/commits/581bca0f2aa873674b5fa30f0f7d308d711c5fe1](url)